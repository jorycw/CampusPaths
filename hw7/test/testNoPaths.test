# Find a path with Dijkstra's algorithm when there are no paths
# between two nodes

CreateGraph graph

AddNode graph a
AddNode graph b

FindPath graph a b