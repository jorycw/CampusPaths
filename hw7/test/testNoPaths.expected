# Find a path with Dijkstra's algorithm when there are no paths
# between two nodes

created graph graph

added node a to graph
added node b to graph

path from a to b:
no path found